### ISSUE TITLE
my latitude/my longitude blocks returns error in chromium 

### ISSUE THREAD

**Issue by gabrielcbe** [NetsBlox/NetsBlox#2647]
Hello, I don't know if here is the best place to post this or if there's a forum I should be posting on but, using the my latitude/my longitude blocks is returning an error when using chromium. 

I've tried it on Ubuntu, Version 79.0.3945.79 (Official Build) Built on Ubuntu , running on Ubuntu 18.04 (64-bit), with error PositionError Network location provider at 'https://www.googleapis.com/' : No response received.
And on Raspbian (Raspebrry Pi), Version 78.0.39.04.108 (Official Build) Built on Raspbian , running on Raspbian 10 32bits, with error, he Network location provider at 'https://www.googleapis.com/' : No response received. 

On Chrome in Windows it seems to be working fine.
The chromium versions are the latest versions available for each OS. 

A couple of months back it used to work and I noticed now they stopped. 
Searching for locations with the Geolocation RPC still works, so example projects can still be used, they just won't load the map when pressing start, only after searching.

Is it something in my end that's wrong or can anyone try to reproduce this error?

Thanks a lot.

**Comment by brollb** [comment]
Hello @gabrielcbe! This is a great place to post issues like this.

I have been able to reproduce the error on chromium. Unfortunately, it appears that this is an issue in chromium's support for the browser geolocation API. (This can be confirmed by visiting other websites using the browser's geolocation API such as https://codepen.io/leemark/pen/ezhlD.)

Are you able to install Google Chrome as opposed to Chromium?

As far as the example projects go, it would probably be good for them to be able to work even if the user does not give access the website access to the user's location (or if the geolocation API is broken as it appears to be in Chromium).

**Comment by gabrielcbe** [comment]
Hello @brollb, thanks for the reply.
Yeah, seems to be a problem only in chromium.
I've downloaded the Chrome on my pc and it works normally. But the schools i'm writing the lesson plans on use Raspberries Pi as computers and in them there's no easy way to have chrome instead of chromium.
I'll do some research to see if anyone fixed this for chromium somehow, otherwise i'll have to wait for a fix in a later chromium version and ask the students to use the search option to get their work done for now.

**Comment by brollb** [comment]
Is the problem only with the example projects? I mean, do the students need to use the "my latitude/longitude" blocks themselves for anything? If not, then the fix could simply be updating the example to work w/o location permissions/capabilities.

**Comment by brollb** [comment]
This appears to be fixed in the latest version of chromium. @gabrielcbe, can you confirm this?

**Comment by gabrielcbe** [comment]
I just tested it on my Raspberry Pi 4 and in there is not yet fixed.
Might be just time now, the latest version on Chromium that I can install there is 84.0.* It takes a while but eventually someone updates the chromium package to run in ARM on Raspberry Pi OS.
I'll burn a new version of the OS to see if it fixes (just to be sure).

p.s.: I tested on Ubuntu with a Chromium in version 87.0.* and it works like a charm.
